### PR TITLE
Feature #286 - Migrate PageColumn component

### LINKED ISSUE #286: Login route: Migrate PageColumn
Migrate `PageMenu`, `PageColumn` and `PageMenuColumn` components from `wallet-demo`.

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will migrate PageColumn component. This component is required for #247.

It will close #286. 

See storybook snapshot:
Screenshot 2019-06-17 16 46 31

### CODE REVIEW

File: packages/medulas-react-components/src/components/Block/index.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import Box from '@material-ui/core/Box';
 import { SizingBreakpoint } from '../Grid';
+import classNames from 'classnames';
+import { Theme, makeStyles } from '@material-ui/core';
 
 export type Unit = 0 | 1 | 2 | 3 | 4;
```
Comment (apanizo) [inline packages/medulas-react-components/src/components/Block/index.tsx]: Can you do a quick check if we have available TS props?

File: packages/medulas-react-components/src/components/Block/index.tsx [hunk 2]
```diff
@@ -75,15 +77,26 @@ interface Props {
...
 
-const Block = ({ children, display = 'block', ...restProps }: Props): JSX.Element => {
+const useStyles = makeStyles((theme: Theme) => ({
+  scroll: {
+    overflowY: 'scroll',
+    overflowX: 'hidden',
+  },
+}));
+
+const Block = ({ children, display = 'block', scroll, className, ...restProps }: Props): JSX.Element => {
```
Comment (apanizo) [inline packages/medulas-react-components/src/components/Block/index.tsx]: To me, a Block is a layout container where it grows as the content grows. If we need to support scroll I would suggest creating a new component. Call it, BoxScroll or something like that. 😉

File: packages/medulas-react-components/src/components/Button/index.tsx [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+
+  return (
+    <MuiButton variant={muiVariant} {...restProps}>
+      {spinner && (
+        <Block marginRight={2}>
+          <CircularProgress size={22} color="inherit" />
+        </Block>
+      )}
+      {children}
+      {variant === 'continue' && <ArrowForwardIcon fontSize="small" />}
```
Comment (apanizo) [inline packages/medulas-react-components/src/components/Button/index.tsx]: Like it.

File: packages/medulas-react-components/src/pages/PageColumn/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+}: Props): JSX.Element => {
+  const classes = useStyles();
+  const theme = useTheme<Theme>();
+
+  const { form, handleSubmit, submitting, invalid, validating } = useForm({
+    onSubmit,
+  });
+
+  return (
+    <Block
```
Comment (apanizo) [inline packages/medulas-react-components/src/pages/PageColumn/index.tsx]: This component is complex, I believe we can simplify it. Some hints:
1. The login page is the only view which has an image on the left, so we can remove the `leftMenu()` and code hre the content.
2. Material ui has a [container](https://material-ui.com/components/container/)  component which can work for the content, you can specify margins automatically.

If you do not have time and you are blocked, just code the first one and create a new ticket for the second one.

File: packages/medulas-react-components/src/utils/storybook/index.tsx [hunk 5]
```diff
@@ -9,27 +9,27 @@ interface Props {
...
+      margin: '0',
+      padding: '0',
+      bottom: '0',
+      top: '0',
+      left: '0',
+      right: '0',
       overflowX: 'hidden',
-      fontFamily: "'Muli', sans-serif",
-      margin: 0,
-      backgroundColor: theme.palette.background.default,
```
Comment (apanizo) [inline packages/medulas-react-components/src/utils/storybook/index.tsx]: Why removing this as well?

File: packages/medulas-react-components/package.json [hunk 6]
```diff
@@ -18,7 +18,9 @@
     "@material-ui/core": "4.0.2",
     "@material-ui/icons": "4.0.1",
     "@material-ui/styles": "4.0.2",
+    "@types/lodash": "^4.14.134",
     "add": "^2.0.6",
+    "lodash": "^4.17.11",
```
Comment (apanizo) [inline packages/medulas-react-components/package.json]: Why lodash? This is a huge library and I prefer to do not include it. Where are you using it?

Comment (albertandrejev) [reply packages/medulas-react-components/package.json]: hmm, you are right, there is no lodash dependencies.

File: packages/medulas-react-components/src/utils/storybook/index.tsx (hunk 5 above)
Comment (albertandrejev) [reply packages/medulas-react-components/src/utils/storybook/index.tsx]: I just get it from `bierzo-wallet` storybook config

File: packages/medulas-react-components/src/components/Button/index.tsx [hunk 7]
```diff
@@ -1,8 +1,30 @@
-import * as React from 'react';
+import React from 'react';
+import { Omit } from '@material-ui/core';
```
Comment (abefernan) [inline packages/medulas-react-components/src/components/Button/index.tsx]: Maybe use Typescript 3.5's Omit.

**Pull request merged**